audit/projects/kubernetes-public/iam.json

```diff
@@ -6,12 +6,6 @@
       ],
       "role": "organizations/758905017065/roles/iam.serviceAccountLister"
     },
-    {
-      "members": [
-        "group:k8s-infra-rbac-slack-infra@kubernetes.io"
-      ],
-      "role": "organizations/758905017065/roles/secretmanager.secretLister"
-    },
```

Comment:
Expected, this was me https://github.com/kubernetes/k8s.io/pull/1800#discussion_r604585782